src/clr-addons/quick-list/add-option.service.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface ClrQuickListValue<T> {
+  id: string;
+  label?: string;
+  value: T;
```

Comment:
question remains: is this value needed? as it's not needed in the whole component

Reply:
This value not needed within the component. I assume it's there to keep tracking of the selected values, the real object behind the the labels themselves.

Reply:
could be done with id also and would be more accurate right? And why do you assume? You wrote the component, so there must be a reason behind it.

Reply:
I think I miss explained my self, so again : 
- we use the ids to track the selected options.
- the values contain the real data behind the ids, we don't use them in the component. but the developer will use them directly, he will not have to check again for the selected ids, and look for the corresponding objects.